mk/appveyor.bat

```diff
@@ -50,7 +50,7 @@ set TARGET=%TARGET_ARCH%-pc-windows-msvc
 %RUSTUP_EXE% -y --default-host %TARGET% --default-toolchain %RUST%
 if %ERRORLEVEL% NEQ 0 exit 1
 
-set PATH="C:\Users\appveyor\.cargo\bin";%cd%\windows_build_tools;%PATH%
+set PATH=%USERPROFILE%\.cargo\bin;%cd%\windows_build_tools;%PATH%
```

Comment:
This seems reasonable to me.

The first line of the commit should mention AppVeyor and should end in a period. 

The commit message should end with the contribution statement from the README.

The commit messages should be wrapped to ~72 characters.